src/components/Pane/Pane.Props.ts

```diff
@@ -0,0 +1,78 @@
+import * as React from 'react';
+import { Pane } from './Pane';
+
+export interface IPaneProps extends React.Props<Pane> {
+    /**
+     * Whether the pane is displayed.
+     * @default false
+     */
+    hidden?: boolean;
+
+    /**
+     * Mode of the pane.
+     * @default PaneMode.push
+     */
+    paneMode?: PaneMode;
+
+    /**
+     * Has the close button visible.
+     * @default true
+     */
+    closeButtonHidden?: boolean;
```

Comment:
Maybe the default here should be false. We've updated the naming guidelines in BESTPRACTICES.md to name the props the opposite of the default value. That way the user can predict the value. Also is consistent with how html attributes are named.

`closeButtonVisible`